### ISSUE TITLE
pitchbender module sends pitchbend messages even if disabled or deleted

### ISSUE THREAD

**Issue by crshrprt** [BespokeSynth/BespokeSynth#1538]
## Describe the bug

pitchbender module sends pitchbend messages  to its destination even if disabled or deleted.

## Build info

Linux, Debian sid, Bespoke Synth nightly

## Steps To Reproduce

Video:

https://github.com/BespokeSynth/BespokeSynth/assets/114294020/566da454-aa9a-4309-9ca1-f8e4ba7ffd64

## Expected vs. actual behavior

pitchbender module should stop sending messages to its destination if disabled or deleted

**Comment by merkys** [comment]
Is this still an issue as of 63be9f82b749ec2d22a2e9e5b11d19efcb562f22? I do not have the said LV2 plugin, but when routing notes from `pitchbender` to an oscillator I can notice disable button affecting the pitch.

**Comment by crshrprt** [comment]
Yes, still a problem. I don't even need a MIDI monitor plugin. I can reproduce it also using `modulationvisualizer`.

https://github.com/user-attachments/assets/02e3dbe8-1c13-477d-ba67-0447e65cde13

**Comment by crshrprt** [comment]
As a workaround for the moment I found 2 ways to stop it:
1. Before deleting the `pitchbender` module I disable it and send another note, it seems to override the pitch modulation
2. Modulating `pitchbender` with an external LFO, when deleting the `pitchbender` module the pitch modulation stops

**Comment by merkys** [comment]
I confirm I get the same result from `modulationvisualizer`. So the bug is still there.
